session/session_test.go

```diff
@@ -3904,9 +3904,7 @@ func (s *testSessionSerialSuite) TestIssue21943(c *C) {
 	c.Assert(err.Error(), Equals, "[variable:1238]Variable 'last_plan_from_cache' is a read only variable")
 }
 
-func (s *testSessionSuite) TestValidateReadOnlyInStalenessTransaction(c *C) {
-	c.Assert(failpoint.Enable("github.com/pingcap/tidb/executor/mockStalenessTxnSchemaVer", "return(false)"), IsNil)
-	defer failpoint.Disable("github.com/pingcap/tidb/executor/mockStalenessTxnSchemaVer")
+func (s *testSessionSerialSuite) TestValidateReadOnlyInStalenessTransaction(c *C) {
```

Comment:
I suggest move this test in `stale_txn_test.go`

Reply:
Maybe next PR, this PR is not for that. And it is already large enough.